packages/react-native/React/CoreModules/RCTTiming.mm

```diff
@@ -128,6 +128,11 @@ - (void)setup
   _paused = YES;
   _timers = [NSMutableDictionary new];
   _inBackground = NO;
+  dispatch_async(dispatch_get_main_queue(), ^{
+    if (!self->_inBackground && [UIApplication sharedApplication].applicationState == UIApplicationStateBackground) {
+      [self appDidMoveToBackground];
+    }
+  });
```

Comment:
We have a couple of utilities which is better to use:
- [`RCTExecuteOnMainQueue`](https://github.com/facebook/react-native/blob/main/packages/react-native/React/Base/RCTUtils.h#L37) rather than `dispatch_async(dispatch_get_main_queue()`
- [`RCTSharedApplication`](https://github.com/facebook/react-native/blob/main/packages/react-native/React/Base/RCTUtils.h#L88) rather than `[UIApplication sharedApplication]`

Reply:
@cipolleschi Done.